docs/pagination.md

```diff
@@ -0,0 +1,92 @@
+## Pagination in Bleve
```

Comment:
Change to single `#` and sub headings accordingly to stay in format with the other markdowns.